Example/Fuse/ViewController.swift

```diff
@@ -94,51 +94,28 @@ class ViewController: UITableViewController {
     }
     
     func filterContentForSearchText(_ searchText: String) {
-        let pattern = fuse.createPattern(from: searchText)
-        
-        typealias ItemTuple = (item: NSMutableAttributedString, score: Double)
-        var results = [ItemTuple]()
-        
         let boldAttrs = [
             NSFontAttributeName : UIFont.boldSystemFont(ofSize: 17),
             NSForegroundColorAttributeName: UIColor.blue
         ]
-        
-        books.forEach {
-            // Search for the pattern in the book
-            if let result = fuse.search(pattern, in: $0) {
-                
-                // if a result is found, do some text transformation so that
-                // the substrings that are matched are bolded in the UI
-                let ranges = result.ranges
-                
-                let attributedString = NSMutableAttributedString(string:"")
-                var index: Int = 0
-                var range = ranges[index]
-                
-                $0.characters.enumerated().forEach { (offset, character) in
-                    let str = String(character)
-                    
-                    if offset > range.upperBound && index < ranges.count - 1 {
-                        index += 1
-                        range = ranges[index]
-                    }
-                    
-                    if offset >= range.lowerBound && offset <= range.upperBound {
-                        attributedString.append(NSMutableAttributedString(string: str, attributes: boldAttrs))
-                    } else {
-                        attributedString.append(NSMutableAttributedString(string: str))
+
+        let results = fuse.search(searchText, in: books)
+
+        filteredBooks = results
+            .sorted { $0.score < $1.score }
```

Comment:
Realized you're using the array search above. So, ignore my previous comment, and this line should be deleted 😄